legal-api/src/legal_api/reports/report.py

```diff
@@ -946,6 +948,11 @@ def _format_continuation_in_data(self, filing):
         # set expro business
         if filing['continuationIn'].get('business'):
             filing['exproBusiness'] = filing['continuationIn']['business']
+            # format founding date
+            founding_date_str = filing['exproBusiness']['foundingDate']
```

Comment:
@severinbeauvais did we include this now in UI? We need to add validation if we are collecting this value

Reply:
Should be `['business']`, ie

![image](https://github.com/user-attachments/assets/283855af-eff8-44ab-8330-19a8e0e38396)

Reply:
> is it ok to use 'exproBusiness' because line 950 set 'exproBusiness'? ![image](https://private-user-images.githubusercontent.com/122495122/380289892-bcaa20e7-4f56-44ab-ab1c-d83bb16a3a30.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mjk4ODQxNDQsIm5iZiI6MTcyOTg4Mzg0NCwicGF0aCI6Ii8xMjI0OTUxMjIvMzgwMjg5ODkyLWJjYWEyMGU3LTRmNTYtNDRhYi1hYjFjLWQ4M2JiMTZhM2EzMC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMDI1JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTAyNVQxOTE3MjRaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT02NTRhZjI4NDRjNTg5ZmM5MDYyN2YzMmEwZWQwZjliNWU1OGE3MTNjZWM5NTM3Mzk4NWI1YjQ0ODVmOWFjN2RkJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.6rVTL1vQiRC7C-fe6ZhRJw9MQRqZxNKs8cVF3twgRuQ)

Yes. I was asking about foundingDate. we don't have a validation for that.

Reply:
Then, yes, we include foundingDate in the UI for an expro, and _someone_ needs to add Legal API validation for that.

What should be checked?
- valid date/time format
- is today or before
- anything else?

Sample filing JSON:
![image](https://github.com/user-attachments/assets/39ba7c98-4bd2-494a-a45b-7f1ae089e6d4)

Reply:
we already validate legal name of expro by fetching from colin, can do the same thing for foundingDate

Reply:
What's the path forward for this -- do it as part of an existing ticket or create a new ticket?

Reply:
create new ticket. we dont have any pending ticket to work on

Reply:
New ticket created #24073
https://app.zenhub.com/workspaces/entities---olga-65af15f59e89f5043c2911f7/issues/gh/bcgov/entity/24073